webtau/src/main/java/com/twosigma/webtau/WebTauDsl.java

```diff
@@ -55,7 +53,7 @@
     }
 
     public static WebTauConfig getCfg() {
-        return WebTauConfig.getInstance();
+        return WebTauConfig.getCfg();
```

Comment:
Do you still need this method here?

Reply:
yes, this one is important, that's how you can do
`cfg.myVarFromConfig` in your tests. I.e. officially exposed cfg access through a single Dsl class